web/src/components/data-fields/non-essential-questions/LocationBasedNonEssentialQuestions.test.tsx

```diff
@@ -17,9 +18,14 @@ import { render, screen } from "@testing-library/react";
 
 jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
 
+jest.mock("@/lib/utils/useIntersectionOnElement", () => ({
+  useIntersectionOnElement: jest.fn(),
+}));
+
 describe("LocationBasedNonEssentialQuestions", () => {
```

Comment:
Had to add the mock for useIntersectionOnElement which happens when the NonEssentialQuestion Component is called